website/www/site/assets/js/quotes-slider.js

```diff
@@ -10,33 +10,32 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-var slider = new KeenSlider("#my-keen-slider", {
-  loop: true,
-  created: function (instance) {
-    var dots_wrapper = document.getElementById("dots");
-    var slides = document.querySelectorAll(".keen-slider__slide");
-    slides.forEach(function (t, idx) {
-      var dot = document.createElement("button");
-      dot.classList.add("dot");
-      dots_wrapper.appendChild(dot);
-      dot.addEventListener("click", function () {
-        instance.moveToSlide(idx);
-      });
-    });
-    updateClasses(instance);
-  },
-  slideChanged(instance) {
-    updateClasses(instance);
+$(document).ready(function(){
+  const SelectorNames = {
```

Comment:
I think using strings without constants like SelectorNames increases readability in this case:

`$('.slick-JS').slick({`
...
`appendDots: '.dots-wrapper',
`dotsClass: 'dots',`